doc/api/util.md

```diff
@@ -564,9 +564,9 @@ terminals.
 
 <!-- type=misc -->
 
-Objects may also define their own `[util.inspect.custom](depth, opts)` function
-that `util.inspect()` will invoke and use the result of when inspecting the
-object:
+Objects may also define their own <code>[[util.inspect.custom][]](depth,
```

Comment:
This is a bit uncommon formatting for our doc, and it baffles our doctools (or rather `marked` module they use). See the rendered page with HTML source:

<details>
<summary>Screenshot:</summary>

![l](https://user-images.githubusercontent.com/10393198/40376268-9a311522-5df6-11e8-9a62-82a30aca43b3.png)

</details><br>

It seems this will do:
```md
[`[util.inspect.custom](depth, opts)`][util.inspect.custom]
```

Reply:
[Done](https://github.com/nodejs/node/pull/20857/commits/ac2591536a32610b52854ab0927f51a34f9df2f6), thanks.